public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx

```diff
@@ -165,6 +166,7 @@ export function DashboardSettings({ dashboard, editview }: Props) {
                 {pages.map((page) => (
                   <Link
                     onClick={() => reportInteraction(`Dashboard settings navigation to ${page.id}`)}
+                    aria-label={selectors.pages.Dashboard.Settings.General.sectionItems(page.title)}
```

Comment:
This would cause screen readers to announce this as "Dashboard settings section item Annotations", "Dashboard settings section item Variables", "Dashboard settings section item Links" etc.

I'm not totally up to date personally on the "running e2e tests on previous versions of grafana" effort, so we'll need to check for what the correct way is.

We can't add in aria-labels for this purpose.

Reply:
Thanks for the explanation 👍 That makes sense!
I'll close this PR and check e2e with data-test-id instead